docs/notebooks/division_and_mod_demonstration.ipynb

```diff
@@ -14,6 +15,16 @@
     }
   },
   "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/N-J-Martin/conjure/blob/fixColabLinks/docs/notebooks/division_and_mod_demonstration.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

Comment:
I think here we want something like:

https://colab.research.google.com/github/conjure-cp/conjure/...

Otherwise it refers to your fork.